en/config/actuators.md

```diff
@@ -0,0 +1,140 @@
+# Actuator Configuration and Testing
+
+:::note
+The *Actuators* view is only displayed if dynamic control allocation is enabled using the [SYS_CTRL_ALLOC](../advanced_config/parameter_reference.md#SYS_CTRL_ALLOC) parameter.
+This replaces geometry and mixer configuration files with configuration using parameters.
+
+You should also ensure that the appropriate airframe type is selected using [CA_AIRFRAME](../advanced_config/parameter_reference.md#CA_AIRFRAME).
+:::
+
+After selecting an [airframe](../config/airframe.md) you will generally need to configure the specific geometry, assign actuators to outputs, and test the actuator response.
+This can be done in *QGroundControl*, under the **Vehicle Setup > Actuators** tab.
+
+The multicopter configuration screen looks like this.
+
+![Actuators MC (QGC)](../../assets/config/actuators/qgc_actuators_mc_aux.png)
+
+Note that some settings are hidden unless you select the **Advanced** checkbox in the top right corner.
+
+## Geometry
+
+The geometry section is used to configure any additional geometry-related settings for the selected [airframe](../config/airframe.md).
```

Comment:
I reworked this section and added a screenshot. Key point here is that geometry is the "expectation". Also that it might display different things for different frames. 

NOTE, I cannot get this image to display myself - i.e. no images of the frame etc.